UnitTests/PayPalUI_Tests/PayPalCreditButton_Tests.swift

```diff
@@ -8,18 +10,27 @@ class PayPalCreditButton_Tests: XCTestCase {
         XCTAssertEqual(payPalCreditButton.imageView?.image, UIImage(named: "PayPalCreditLogo"))
     }
 
-    func testInit_whenPayPalCreditButtonCreated_hasUIColorFromAssets() {
-        let payPalCreditButton = PayPalCreditButton()
-        XCTAssertEqual(payPalCreditButton.containerView.backgroundColor, PaymentButtonColor.darkBlue.color)
+    func testInit_whenPayPalCreditButtonCreated_hasDefaultUIValues() {
+        let sut = PayPalCreditButton()
+        XCTAssertEqual(sut.edges, PaymentButtonEdges.softEdges)
+        XCTAssertEqual(sut.size, PaymentButtonSize.collapsed)
+        XCTAssertEqual(sut.color, PaymentButtonColor.darkBlue)
+        XCTAssertNil(sut.insets)
     }
 
-    func testInit_whenSwiftUIPayPalCreditButtonCreated_canInit() {
-        let action = { }
-        let payPalCreditButton = PayPalCreditButton { }
-        let coordinator = Coordinator(action: action)
+    func testMakeCoordinator_whenPayPalCreditButtonRepresentableIsCreated_actionIsSetInCoordinator() {
+        let expectation = expectation(description: "Action is called")
+        let sut = PayPalCreditButton.Representable {
+            expectation.fulfill()
+        }
+        let coordinator = sut.makeCoordinator()
 
-        XCTAssertNotNil(payPalCreditButton)
-        XCTAssertNotNil(payPalCreditButton.makeCoordinator())
-        XCTAssertNotNil(coordinator.onAction(action))
+        coordinator.onAction(self)
+        XCTAssertNotNil(sut)
+        waitForExpectations(timeout: 1) {error in
```

Comment:
I've not used `waitForExpectations` in this way before. Is there an advantage to capturing the error here vs `waitForExpectations(timeout: 1)`?

Reply:
@jaxdesmarais well not in this case. It might help if we wanted to return an error with a specific message, or maybe continue with the test even if there was a failure. I removed it because it doesnt serve any purpose here

Reply:
I think the main benefit to the way Jose had it was it lets us show a custom failure message if `waitForExpectations` isn't called. I think this is most helpful when CI fails, we get a more detailed reason why.